### ISSUE TITLE
[A11y]: Icon in disclaimer window has no accessible text

### ISSUE THREAD

**Issue by jclintonibm** [carbon-design-system/carbon-ai-chat#876]
### Browser

Chrome

### Package version

1.2.1

### React version (if applicable)

18.2.0

### Automated testing tool and ruleset

IBM Equal Access toolkit 1.1.1

### Description

Impedes use by customers with accessibility needs and impacts Federal compliance.

<img width="1909" height="831" alt="Image" src="https://github.com/user-attachments/assets/932baf61-5ed5-4383-8b8f-658db739cc01" />

### WCAG 2.2 violation

1.1.1 accessible text

### Reproduction / example

None

### Steps to reproduce

Create a new react Chat component with disclaimer specified.

### Suggested severity

Severity 2 = User cannot complete task, and/or no workaround within the user experience of a given component.

### Code of conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon-ai-chat/issues) for duplicate problems
